src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -1770,6 +1779,28 @@ public NamedPipeConnectionInfo()
             OpenTimeout = openTimeout;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NamedPipeConnectionInfo"/> class.
+        /// </summary>
+        /// <param name="debugPipeName">Pipe name to connect to.</param>
+        public NamedPipeConnectionInfo(
+            string debugPipeName) :
+            this(debugPipeName, _defaultOpenTimeout)
+        { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NamedPipeConnectionInfo"/> class.
+        /// </summary>
+        /// <param name="debugPipeName">Pipe name to connect to.</param>
+        /// <param name="openTimeout">Open time out in Milliseconds.</param>
+        public NamedPipeConnectionInfo(
+            string debugPipeName,
+            int openTimeout)
+        {
+            DebugPipeName = debugPipeName;
```

Comment:
We should validate that `debugPipeName` parameter is not null or empty.

Reply:
added.